packages/lookit-initjspsych/src/types.ts

```diff
@@ -1,17 +1,17 @@
+import { JsPsychExpData } from "@lookit/data/dist/types";
 import { DataCollection } from "jspsych";
 
-export type UserFunc = (data: DataCollection) => void;
+export type UserFuncOnDataUpdate = (data: JsPsychExpData) => void;
+export type UserFuncOnFinish = (data: DataCollection) => void;
 
-export type ResponseData = {
-  id: string;
-  type: "responses";
-  attributes: {
-    exp_data: DataCollection[];
-    completed?: boolean;
-  };
+export type JsPsychOptions = {
+  on_data_update?: UserFuncOnDataUpdate;
+  on_finish?: UserFuncOnFinish;
 };
 
-export type JsPsychOptions = {
-  on_data_update?: UserFunc;
-  on_finish?: UserFunc;
+export type Timeline = {
+  type: {
+    chsData?: () => object;
+  };
+  data?: object;
```

Comment:
I was initially really confused by this type, but I think I get it now... Just to confirm:
This is the type for the trial objects inside the array that's passed to `jsPsych.run`? And each trial object must have a `type` (the plugin class name), which may or may not have a `chsData` static method on the plugin class.
These objects may or may not have a `data` property that is defined by the researcher.
And of course they can have a bunch of other properties (parameters) that aren't listed here. 
Is that right?

Reply:
You're correct.  The actual jsPsych type for a timeline object isn't exported.  Or at least, I couldn't find where it was. I would rather use the types provided by jsPsych.